list.yaml

```diff
@@ -36,6 +36,13 @@
 
 
 # A
+- name: "ABlog"
+  github: "abakan/ablog"
+  license: license.txt "Permission is hereby granted, free of charge (...)"
+  website: "http://ablog.readthedocs.org/"
+  language: "Python"
+  description: "ABlog is a Sphinx extension that converts any documentation or personal website project into a full-fledged blog with"
```

Comment:
I think, the information, that it is for Sphinx and it is Python based is important. The others also have this information.